.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+## Bug Description
+<!--
+Please provide a clear and concise description of what the bug is.
+
+What did you expect to happen? What actually happened?
+
+For bugs with fixers: How the code was fixed? How did you expect the code to be fixed?
```

Comment:
I think this might be better placed in the `code snippet` section.
Also: "_How **was** the code ~~was~~ fixed_"